code-matlab/shared/datatypes/iv/TSDtoIV.m

```diff
@@ -54,7 +54,7 @@
 switch cfg.method
     case 'zscore'
         [temp_data,mu,sigma] = zscore(temp_data);
-        switch cfg.dcn
+        switch cfg.operation
```

Comment:
this is better than the previous version, but are there any dependencies, i.e. does some code use the .dcn version? can there be some kind of compatibility check that will still accept .dcn?

Reply:
To be honest, this was a fix to make it consistent within the function. It already switched over to "operation" from "dcn" but I don't think a compatibility check was implemented when this change was first introduced in commit 1ce4b2b6351a5266dcf4160b27bbfc019fee5bf6. I can implement the check though...